modules/codegen-plugin/src/sbt-test/codegen-plugin/aws-newtype-flatten/test

```diff
@@ -0,0 +1,8 @@
+# check if smithy4sCodegen works
+> compile
```

Comment:
let's change it to only generate the sources : the dynamodb specs are huge and compiling just for the sake of checking whether the file exists is detrimental to build times

Reply:
Switched to using a lightweight dynamo spec in 578d114